main/perl-math-basecnv/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+pkgname=perl-math-basecnv
+pkgver=1.14
+pkgrel=0
+pkgdesc="Math::BaseCnv perl module"
+url="http://search.cpan.org/dist/Math-BaseCnv/"
+arch="noarch"
+license="GPL-3.0"
+depends="perl"
+makedepends="perl-dev"
+install=""
+subpackages="$pkgname-doc"
+source="http://search.cpan.org/CPAN/authors/id/P/PI/PIP/Math-BaseCnv-$pkgver.tgz"
+
+_builddir="$srcdir"/Math-BaseCnv-$pkgver
+prepare() {
```

Comment:
`default_prepare` handles applying patches now. Options can be supplied through `$patch_args`.